src/debug_clang.cc

```diff
@@ -463,6 +463,7 @@ void Debug::Clang::remove_breakpoint(const boost::filesystem::path &file_path, i
             auto file_spec=line_entry.GetFileSpec();
             boost::filesystem::path breakpoint_path=file_spec.GetDirectory();
             breakpoint_path/=file_spec.GetFilename();
+            breakpoint_path = boost::filesystem::canonical(breakpoint_path);
```

Comment:
And then...? What should be the appropriate response if the path handed out by LLDB doesn't exist? Is it because they got it wrong? Is it because someone removed it in the TOCTTU fashion? Should we crash? I suppose not. Should we ignore it? Sure, but then what is the state of the program? After all, the highlighting is removed by GUI somewhere else, so we can't just log it and move on. I ask because I seriously don't know.